lib/ddtrace/contrib/rspec/ext.rb

```diff
@@ -0,0 +1,19 @@
+module Datadog
+  module Contrib
+    module RSpec
+      # RSpec integration constants
+      module Ext
+        APP = 'rspec'.freeze
+        ENV_ANALYTICS_ENABLED = 'DD_TRACE_RSPEC_ANALYTICS_ENABLED'.freeze
+        ENV_ANALYTICS_SAMPLE_RATE = 'DD_TRACE_RSPEC_ANALYTICS_SAMPLE_RATE'.freeze
+        ENV_ENABLED = 'DD_TRACE_RSPEC_ENABLED'.freeze
+        ENV_OPERATION_NAME = 'DD_TRACE_RSPEC_OPERATION_NAME'.freeze
+        FRAMEWORK = 'rspec'.freeze
+        OPERATION_NAME = 'rspec.it'.freeze
+        EXAMPLE_GROUP_OPERATION_NAME = 'rspec.example'.freeze
```

Comment:
To match rspec nomenclature, I suggest we name these:
```suggestion
        OPERATION_NAME = 'rspec.example'.freeze
        EXAMPLE_GROUP_OPERATION_NAME = 'rspec.example_group'.freeze
```

These would follow rspec's two main structures:

* example: https://github.com/rspec/rspec-core/blob/5d87c28bb828ba14b54a0d8cfddf044894009314/lib/rspec/core/example_group.rb#L145
* example_group: https://github.com/rspec/rspec-core/blob/5d87c28bb828ba14b54a0d8cfddf044894009314/lib/rspec/core/example_group.rb#L246

There's also a top-level concept called "nested_shared_group", but I'm not sure these will look different from an example group at runtime: https://github.com/rspec/rspec-core/blob/5d87c28bb828ba14b54a0d8cfddf044894009314/lib/rspec/core/example_group.rb#L317